doc/api-extensions.md

```diff
@@ -607,3 +607,12 @@ This adds the following new endpoint (see [RESTful API](rest-api.md) for details
 * `PUT /1.0/storage-pools/<pool>/volumes/<type>/<volume>/snapshots/<name>`
 * `POST /1.0/storage-pools/<pool>/volumes/<type>/<volume>/snapshots/<name>`
 * `DELETE /1.0/storage-pools/<pool>/volumes/<type>/<volume>/snapshots/<name>`
+
+## storage\_unmapped
+Introduces a new `security.unmapped` boolean on storage volumes.
```

Comment:
`security.storage.unmapped` maybe?

Reply:
Why? it's a property on a storage volume, so can't really be anything but storage :)

Reply:
I thought the scoping might make it easier to tell what that property is for and what it applies to and also makes it easier when we add keys later. :)

Reply:
Still feels redundant and un-needed when scoped to a custom storage volume config key.

I thought of calling it `security.idmap.unmapped` before to better line up with containers, but the problem with that approach is that `security.unmapped` will still be relevant even when `security.idmap` isn't (shiftfs) as it'll then mean to not use shiftfs keeping the disk unmapped.